Formula/g/gnu-cobol.rb

```diff
@@ -20,17 +21,32 @@ class GnuCobol < Formula
     sha256 x86_64_linux:   "4bf957f586d155f1034ac90ac24edff1253e9de935336680689d85f0bbdcf45c"
   end
 
+  head do
+    # there's no offical git mirror, but this one is quite stable and up to date
+    # consider to use "trunk" - but that would be incompatible to 3.x
+    url "https://github.com/OCamlPro/gnucobol.git", branch: "gnucobol-3.x"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+    depends_on "gettext" => :build
+    depends_on "help2man" => :build
+    depends_on "libtool" => :build
+    depends_on "texinfo" => :build
+  end
```

Comment:
We should just leave this out then

Reply:
What now? Note that this would still build GnuCOBOL in the 3.x branch which is today the same like the release, but will soon be newer.
Should I still get rid of that? Completely or comment out?

Reply:
If we don't have a git head, it should just be removed completely.

Reply:
Or it can be

```ruby
head "https://svn.code.sf.net/p/gnucobol/code/trunk"
```

If that makes sense.

Reply:
Ah, so SVN works, too?

Should this be unstable trunk's "head" (not sure if this builds on homebrew at all) or stable 3.x's "head" [once trunk is stabilized this would be changed, of course]?

Reply:
> Ah, so SVN works, too?

Yes, there are several other formulae that use SVN head.

> Should this be unstable trunk's "head" (not sure if this builds on homebrew at all) or stable 3.x's "head" [once trunk is stabilized this would be changed, of course]?

`trunk` makes more sense to me. `head` is [meant for cutting-edge source](https://docs.brew.sh/Formula-Cookbook#unstable-versions-head).